src/install.js

```diff
@@ -1,19 +1,32 @@
 /* eslint-env node */
 
+/**
+ * @param {import('@quasar/app-vite').InstallAPI} api
+ */
 module.exports = function (api) {
   // Quasar compatibility check.
   api.compatibleWith('quasar', '^2.0.0')
   if (api.hasVite === true) {
     api.compatibleWith('@quasar/app-vite', '^1.0.0')
   } else {
-    api.compatibleWith('@quasar/app', '^3.0.0')
+    api.compatibleWith('@quasar/app-webpack', '^3.3.3')
   }
 
-  // Render templates in the app.
-  if (api.prompts.typescript) {
-    api.render('./templates')
-  } else {
-    api.render('../lib/templates')
+  const hasSubscriptions = api.prompts.subscriptions === true
+
+  api.render('./templates/base')
+  const hasTypescript = api.prompts.typescript === true
+  api.render(`./templates/${hasTypescript ? 'typescript' : 'no-typescript'}`, {
+    hasVite: api.hasVite,
+    hasSubscriptions
+  })
+
+  if (hasSubscriptions) {
+    api.extendPackageJson({
+      dependencies: {
+        'graphql-ws': '^5.14.0'
```

Comment:
I'd use this helper https://github.com/quasarframework/quasar-testing/blob/dev/packages/e2e-cypress/src/install.js#L41-L55 to automatically install the same package version used into the AE devDeps